sdl_android_lib/src/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -282,16 +282,21 @@ public void onProtocolSessionStarted(SessionType sessionType,
 			updateBroadcastIntent(sendIntent, "COMMENT2", " SessionType: " + sessionType.getName());
 			sendBroadcastIntent(sendIntent);	
 			
-			setWiProVersion(version);
-
-			 if ( (_transportConfig.getHeartBeatTimeout() != Integer.MAX_VALUE) && (version > 2) )
-			 {
-				 HeartbeatMonitor heartbeatMonitor = new HeartbeatMonitor();
-				 heartbeatMonitor.setInterval(_transportConfig.getHeartBeatTimeout());
-	             sdlSession.setHeartbeatMonitor(heartbeatMonitor);
-			 }			
+			setWiProVersion(version);	
 			
-			if (sessionType.eq(SessionType.RPC)) {			
+			if (sessionType.eq(SessionType.RPC)) {	
+
+				 if ( (_transportConfig.getHeartBeatTimeout() != Integer.MAX_VALUE) && (version > 2))

```

Comment:
On Core we use 0 to indicate not using heartbeat, is it possible to align with that paradigm?